src/main/java/org/gridsuite/study/server/service/StudyService.java

```diff
@@ -964,15 +971,14 @@ private LoadFlowParametersEntity createParametersEntity(LoadFlowParametersValues
         return LoadFlowService.toEntity(allCommonValues, allSpecificValues);
     }
 
-    public SecurityAnalysisParametersValues getSecurityAnalysisParametersValues(UUID studyUuid) {
-        return studyRepository.findById(studyUuid)
-                .map(studyEntity -> studyEntity.getSecurityAnalysisParameters() != null ? SecurityAnalysisService.fromEntity(studyEntity.getSecurityAnalysisParameters()) : SecurityAnalysisService.getDefaultSecurityAnalysisParametersValues())
-                .orElse(null);
+    public String getSecurityAnalysisParametersValues(UUID studyUuid) {
+        StudyEntity studyEntity = studyRepository.findById(studyUuid).orElseThrow(() -> new StudyException(STUDY_NOT_FOUND));
+        UUID securityAnalysisParametersUuid = securityAnalysisService.getSecurityAnalysisParametersUuidOrElseCreateDefaults(studyEntity);
```

Comment:
Maybe, inside sa server, just return the created object then you don't have to do a second call right after ?

Reply:
As discussed with Ayoub, the case is not supposed to happen so let's keep it like that